load.php

```diff
@@ -287,6 +287,7 @@ function perflab_get_standalone_plugins_constants() {
 		'images/dominant-color-images' => 'DOMINANT_COLOR_IMAGES_VERSION',
 		'images/fetchpriority'         => 'FETCHPRIORITY_VERSION',
 		'images/webp-uploads'          => 'WEBP_UPLOADS_VERSION',
+		'database/sqlite'              => 'SQLITE_MAIN_FILE',
```

Comment:
@felixarntz @mukeshpanchal27 I added this so that the PL SQLite module option can not be accidentally enabled by the end user once the standalone plugin is activated, thus preventing the following unnecessary fatal error:

```
Fatal error: Cannot redeclare w_install (previously declared in /var/www/html/w-content/plugins/sqlite-database-integration/wp-includes/sqlite/install-functions.php:138)in /var/www/html/wp-content/plugins/performance/modules/database/sqlite/wp-includes/sqlite/install-functions.php on line 122
```

Reply:
Great catch!